PHP-Variables.md

```diff
@@ -0,0 +1,164 @@
+# PHP : Variables
+
+Variables are containers for storing data such as ```strings```, ```integers```, ```boolean``` values, ```array``` and objects.
+
+PHP follows certain rules for variable declarations such as:
+
+ * The variable must begin with a dollar sign ($)
+ 
+      Example: 
+  ```javascript
+<?php
+     $var = 5;
+?>
+```
+ * The variable name can contain characters such as A-Z, a-z, 0-9, _ and ASCII characters from 127-255.
+ * [To know more about ASCII](http://www.asciitable.com/ "ASCII Table")
+
+ 
+    Example:
+  ```javascript
+<?php
+     $var = 5; //Valid
+     $var_1 = "Foo"; //Valid
+     $_var2 = 'Bar'; //Valid
+     $var.3 = 'Baz'; //Invalid
+?>
+```
+ * The variable name can begin with underscore (_).
+  
+  Example:
+  ```javascript
+<?php
+     $_var2 = 'Bar'; //Valid
+?>
+```
+ * The variable name must not begin with a number 0-9.
+    Example:
+  ```javascript
+<?php
+     $9var3 = 'Baz'; //Invalid
+?>
+```
+* The variable name is case sensitive.
```

Comment:
check list indentation is not proper in the markdown. this item got pushed into a sub item.